CHANGELOG.md

```diff
@@ -1,5 +1,83 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.14.19
+
+<!-- generated comparing v5.14.18..master -->
+
+_Nov 28, 2023_
+
+A big thanks to the 15 contributors who made this release possible.
+
+### `@mui/material@5.14.19`
+
+- &#8203;<!-- 44 -->[Autocomplete] Fix behavior when there are duplicate labels (#36426) @islandryu
```

Comment:
This could be the highlight of the release.

Reply:
Added it